src/install/installer.ts

```diff
@@ -41,6 +41,7 @@ export async function installBrowsersWithProgressBar(packagePath: string) {
 
   const browsersPath = browserPaths.browsersPath(packagePath);
   await fsMkdirAsync(browsersPath, { recursive: true });
+  const lockfilePath = path.join(browsersPath, '__dirlock');
   const releaseLock = await lockfile.lock(browsersPath, {
```

Comment:
From what I see on 'proper-lockfile' usage, `.lock()` may also throw, as well as `releaseLock`. Should we capture them and rethrow with a nicer error as well?

Reply:
My understanding is that `onCompromised` is called on somewhat expected errors. Like when another process won't let go of our lock. Unexpected errors, like we tried to release the same lock twice, just get thrown as usual.